extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -93,13 +106,28 @@ public void initialize() {
                 updateStatus(statusInfo.getStatus(), statusInfo.getStatusDetail(), statusInfo.getDescription());
                 FullLight fullLight = getLight();
                 if (fullLight != null) {
+                    String modelId = fullLight.getModelID().replaceAll(NORMALIZE_ID_REGEX, "_");
+                    updateProperty(Thing.PROPERTY_MODEL_ID, modelId);
```

Comment:
> Yes, it is already a bug that there is another such call (introduced by the firmware stuff). So far it was forbidden for bindings to add properties themselves.

Where did we define that it is not allowed for bindings to add properties themselves? Especially properties that cannot be declared in the thing type defintion, e.g. serial number, MAC address, firmware version,  have to be added in the handler....